client/plots/matrix.interactivity.js

```diff
@@ -188,9 +191,11 @@ export function setInteractivity(self) {
 					const menuDiv = self.dom.clickMenu.d
 						.append('div')
 						.attr('class', 'sja_menuoption sja_sharp_border')
-						.text(`${s.controlLabels.Sample} summary: ${sampleData._SAMPLENAME_}`)
+						.html(
+							`${s.controlLabels.Sample} summary: <span class='linkText'>${sampleData._SAMPLENAME_} ${svgIcons.externalLink}</span>`
```

Comment:
do not show word summary. it's what gdc calls and not universal
just show `Case: [link]`